content/02.introduction.md

```diff
@@ -47,20 +47,20 @@ archaeologists, museum curators, and scientists can also contribute 'modern'
 contamination. While these characteristics can help provide evidence towards the
 'authenticity' of true aDNA sequences (e.g. aDNA C>T 'damage' profiles
 [@doi:10.1093/bioinformatics/btt193]), they also pose specific challenges such
-as unspecific DNA alignment and/or low coverage and miscoding lesions resulting
+as unspecific DNA alignment and/or low coverage and miscoding lesions that can result 
 in low-confidence genotyping. These factors often lead to prohibitive sequencing
-costs to retrieve enough data for modern NGS data pipelines (e.g. > 1 billion
+costs to retrieve enough data for modern NGS data pipelines (such as > 1 billion
 reads for a 1X depth coverage _Yersinia pestis_ genome
-[@doi:10.1016/j.cell.2015.10.009]), and thus require aDNA tailored methods and
+[@doi:10.1016/j.cell.2015.10.009]), and thus require aDNA-tailored methods and
 techniques to overcome these challenges.
 
 Two previously published and commonly used pipelines in the field are PALEOMIX
 [@doi:10.1038/nprot.2014.063] and EAGER [@doi:10.1186/s13059-016-0918-z]. These
 two pipelines take a similar approach to link together standard tools used for
 Illumina NGS data processing (sequencing quality control, sequencing adapter
-removal and/or paired-end read merging, mapping of reads to a reference genome,
-genotyping, etc.), but with a specific focus on tools that are designed for or
-well-suited for aDNA (such as bwa aln that works well on ultra-short molecules
+removal/and or paired-end read merging, mapping of reads to a reference genome,
```

Comment:
Reverting correction to a typo

Reply:
I think that's fixed in your next one, so will leave this here.